### PR TITLE
Alltrans

### PR DESCRIPTION
**Pull request by IsraaSulaiman**
#38 #39

### CODE REVIEW

File: public/css/allTransactions.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
```
Comment (engshorouq) [inline public/css/allTransactions.css]: already exists in main.css

File: public/css/error.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+}
+
+.main {
+    min-height: 200px;
+    text-align: center;
+}
```
Comment (engshorouq) [inline public/css/error.css]: add line here

File: src/controllers/error.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+exports.server = (req, res) => {
```
Comment (engshorouq) [inline src/controllers/error.js]: also server have 5 parameters ` error, req, res, next `

File: src/views/allTransactions.hbs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            <td>{{name}}</td>
+            {{#if description}}
+            <td>{{description}}</td>
+            {{else}}
+            <td>-<td>
+            {{/if}}
+        </tr>
+        {{/each}}
+    </table>
+</div>
```
Comment (engshorouq) [inline src/views/allTransactions.hbs]: add line here

File: src/views/error.hbs [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<section>
+    <h1>{{status}} || {{errorMsg}} </h1>
+</section>
```
Comment (engshorouq) [inline src/views/error.hbs]: add line here

File: src/views/layouts/error.hbs [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+<body>
+    <div class='wrapper'>
+        {{>header}}
+        <main class='main'>
+            {{{body}}}
+        </main>
+        {{>footer}}
+    </div>
+</body>
+</html>
```
Comment (engshorouq) [inline src/views/layouts/error.hbs]: add line here

**Review by asmaa-ezz — APPROVED** [review]

**Pull request closed without merging**